docs/usage/rtk-query/cached-data.md

```diff
@@ -0,0 +1,516 @@
+---
+id: cached-data
+title: Cached Data
+sidebar_label: Cached Data
+hide_title: true
+---
+
+# Cached Data
+
+A key feature of RTK Query is it's management of cached data. When data is fetched from the server, RTK Query will store the data in the redux store as a 'cache'. When an additional request is performed for the same data, RTK Query will provide the cached data rather than sending an additional request to the server.
+
+RTK Query provides a number of concepts and tools to manipulate the cache behaviour and adjust it to your needs.
+
+## Definitions
+
+- _tags_ - Items used to identify the data present in the cache. An individual `tag` has a `type`, represented as a `string` name, and an optional `id`, represented as a `string` or `number`.
+- _providing tags_ - Used to advise which tags are provided by an individual query endpoint, and can be considered 'valid' in the cache
+- _invalidating tags_ - Identifies which tags are invalidated by an individual mutation endpoint, and should no longer be considered 'valid' in the cache
+
+  TODO: description of any other key concepts
+
+## Default cache handling behaviour
+
+TODO:
+
+## Cache tags
+
+RTK Query uses the concept of 'tags' to determine whether a mutation for one endpoint intends to 'invalidate' some data that was 'provided' by a query from another endpoint. When defining an `api`, `createApi` accepts an array of tag type names for the `tagTypes` property, which is a list of possible tag name options that the queries for the `api` could provide.
```

Comment:
```suggestion
RTK Query uses the concept of 'tags' to determine whether a mutation for one endpoint intends to 'invalidate' some data that was 'provided' by a query from another endpoint.
If cache data is being invalidated, it will either refetch the providing query (if components are still using that data) or remove the data from the cache.

When defining an `api`, `createApi` accepts an array of tag type names for the `tagTypes` property, which is a list of possible tag name options that the queries for the `api` could provide.
```